apps/backend/core/configuration/__init__.py

```
@@ -0,0 +1,185 @@
+    ServicePorts,
+    ServiceTimeouts,
+)
+from apps.backend.core.configuration.services import ServiceEndpoints
+
+# ========== Enhanced Configuration Classes ==========
+
+# Extend BaseConfig to include services and constants
+# This is done via monkey-patching to avoid circular imports
+
+
+def _enhance_config_classes():
```

Ruff (0.14.4)

Missing return type annotation for private function `_enhance_config_classes`

Add return type annotation: `None`

(ANN202)